services/date/date.service.js

```diff
@@ -11,16 +11,18 @@ export default class Date extends BaseService {
   static category = 'other'
   static route = { base: 'date', pattern: ':timestamp(-?[0-9]+)' }
 
-  static examples = [
-    {
-      title: 'Relative date',
-      pattern: ':timestamp',
-      namedParams: { timestamp: '1540814400' },
-      staticPreview: this.render({ relativeDateString: '2 days ago' }),
-      keywords: ['time', 'countdown', 'countup', 'moment'],
-      documentation,
+  static openApi = {
+    '/date/{timestamp}': {
```

Comment:
I'm just now realizing (or perhaps have just forgotten) that I'm not sure if/how we're doing any parameter validation on the frontend badge builder in the new ui :thinking: 

More of a reminder/edification question, definitely non-blocking

Reply:
I think this is the relevant bit of validation:

https://github.com/badges/shields/blob/9118ba8d24d8abc0de723a3e8a5cb36020248fbc/core/base-service/base.js#L214-L229

Added in #9320 in response to https://github.com/badges/shields/pull/9320#discussion_r1249935938